src/css/components/_captions-settings.scss

```diff
@@ -102,4 +117,5 @@
 
 .vjs-track-settings-controls .vjs-default-button {
   margin-right: 1em;
+  margin-bottom: 1em;
```

Comment:
the bottom buttons appear right on the border of the dialog box for small players, this fixes that.